src/app/containers/CpsRecommendations/index.jsx

```diff
@@ -1,58 +1,62 @@
 import React, { useContext } from 'react';
-import { arrayOf, shape, number } from 'prop-types';
+import { arrayOf, shape } from 'prop-types';
 import styled from '@emotion/styled';
 import {
   GEL_GROUP_2_SCREEN_WIDTH_MIN,
-  GEL_GROUP_3_SCREEN_WIDTH_MAX,
   GEL_GROUP_3_SCREEN_WIDTH_MIN,
   GEL_GROUP_4_SCREEN_WIDTH_MIN,
 } from '@bbc/gel-foundations/breakpoints';
 import pathOr from 'ramda/src/pathOr';
 import path from 'ramda/src/path';
-import { C_LUNAR } from '@bbc/psammead-styles/colours';
+import { C_GHOST } from '@bbc/psammead-styles/colours';
 import {
   GEL_SPACING,
   GEL_SPACING_DBL,
   GEL_SPACING_TRPL,
 } from '@bbc/gel-foundations/spacings';
 import SectionLabel from '@bbc/psammead-section-label';
+import SkipLinkWrapper from '#components/SkipLinkWrapper';
 
 import { storyItem } from '#models/propTypes/storyItem';
 import { ServiceContext } from '#contexts/ServiceContext';
 import useToggle from '#hooks/useToggle';
-import CpsOnwardJourney from '../CpsOnwardJourney';
 import { GridItemMediumNoMargin } from '#app/components/Grid';
-import RecommendationsPromo from './RecommendationsPromo';
 import RecommendationsPromoList from './RecommendationsPromoList';
+import RecommendationsPromo from './RecommendationsPromo';
 
 const RecommendationsWrapper = styled.div`
-  background-color: ${C_LUNAR};
+  background-color: ${C_GHOST};
   margin: ${GEL_SPACING_TRPL} 0;
   padding: ${GEL_SPACING_DBL} ${GEL_SPACING};
-  @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_3_SCREEN_WIDTH_MAX}) {
+  @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
     margin: 0 0 ${GEL_SPACING_TRPL};
-    padding: 0 0 ${GEL_SPACING_DBL};
+    padding: 0 ${GEL_SPACING_DBL} ${GEL_SPACING_DBL};
   }
-  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+  @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
     margin: 0 0 ${GEL_SPACING_TRPL};
     padding: ${GEL_SPACING_DBL} 0;
   }
-  @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
-    padding: ${GEL_SPACING_DBL};
-  }
 `;
 
 const LabelComponent = styled(SectionLabel)`
   margin: 0;
   padding: 0;
   @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
-    margin: 0 0 ${GEL_SPACING_DBL};
+    margin: 0 0 ${GEL_SPACING};
   }
 `;
 
-const CpsRecommendations = ({ items, parentColumns }) => {
-  const { recommendations, translations } = useContext(ServiceContext);
+const CpsRecommendations = ({ items }) => {
```

Comment:
Is `items` guaranteed to be a non-empty array at this point?

Reply:
No, the check is done on line 63